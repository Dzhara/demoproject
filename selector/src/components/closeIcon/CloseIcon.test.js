import React from "react";
import renderer from "react-test-renderer";
import CloseIcon from "./CloseIcon";

describe("CloseIcon", () => {
  test("snapshot renders", () => {
    const component = renderer.create(
      <CloseIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
