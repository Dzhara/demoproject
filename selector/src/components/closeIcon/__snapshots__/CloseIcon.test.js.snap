// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon snapshot renders 1`] = `
<div
  className="close"
  onClick={[Function]}
/>
`;
